hawkular-nest/hawkular-nest-itest/pom.xml

```diff
@@ -290,23 +294,34 @@
       </plugin>
 
       <plugin>
-        <artifactId>maven-resources-plugin</artifactId>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>copy-resources</id>
+            <id>create-itest-users</id>
```

Comment:
> As I said, first, to ensure that we do not rely on hard coded defaults too much

Ok, now I see what you mean there. 

I'm not sure I see the benefit, to be honest. I usually prefer to have the test to be explicit, ie, setting the username/password itself, rather then getting this value from "somewhere" (the pom or sysprop, in this case).

But I guess this is a matter of taste, so, I'm with this change.

> And having random passwords makes the test system less vulnerable :)

:)